src/components/Layout/Layout.js

```diff
@@ -0,0 +1,72 @@
+import React from "react";
+import Grid from "@material-ui/core/Grid";
+import { makeStyles } from "@material-ui/core/styles";
+import Paper from "@material-ui/core/Paper";
+import Hidden from "@material-ui/core/Hidden";
+import LabelBottomNavigation from "./BottomNav/BottomNav";
+import HomePage from "../HomePage/HomePage";
+import SideNavigation from "./SideNav/SideNav";
+
+const useStyles = makeStyles((theme) => ({
+  root: {
+    flexGrow: 1,
+  },
+  paper: {
+    height: "40rem",
+    width: "80%",
+    position: "fixed",
+    marginRight: "2rem",
+  },
+  paper1: {
+    height: "100%",
+    position: "fixed",
+  },
+}));
+
+const Layout = (props) => {
+  const classes = useStyles();
+  const wrapper = React.createRef();
```

Comment:
seems like you don't need this

Reply:
@rowlandekemezie sir, the "const wrapper = React.createRef()" was added because I was receiving warnings of findDOM deprecation, and I was asked to add the ref attribute to the node.